src/System.Collections/tests/Generic/Queue/Queue.Generic.Tests.cs

```diff
@@ -149,6 +149,30 @@ public void Queue_Generic_EnqueueAndDequeue(int capacity, int items)
             Assert.Equal(itemToAdd, q.Dequeue());
         }
 
+#if netcoreapp11
```

Comment:
My preference would be to have these netcoreapp11 specific tests in their own file instead of using #ifdefs.